apicast/src/policy/local_chain.lua

```diff
@@ -0,0 +1,36 @@
+local policy = require('policy')
+local Proxy = require('proxy')
+local _M = policy.new('Local Policy Chain')
+
+local default_chain = require('policy_chain').build()
```

Comment:
There's no need to call `.build()` here. It's already done in the `policy_chain` module.

Reply:
Yeah I was hoping for that too, but this fails somehow. My guess is that the apicast module has some state and needs one copy in the memory to be used in both access and post_action. 
You can see the failing test when this is nil. But probably there are better ways to solve this. Just something to keep in mind.

Reply:
I'd call `.build()` with `{ 'apicast' }`. I know it's the default, but I think making it explicit makes the code easier to follow.
What do you think?

Reply:
Yep. Possibly even removing the `{ 'apicast' }` default and doing it explicitly everywhere (hopefully just one place).